apps/src/sites/studio/pages/lessons/edit.js

```diff
@@ -53,6 +56,10 @@ $(document).ready(function() {
     });
   });
 
+  if (activities.length === 0) {
+    activities = emptyActivity;
```

Comment:
this looks like a type mismatch -- activities is an array but emptyActivity is a single activity. how about `activities.push(emptyActivity)`? That'd let you switch `let activity` back to `const` earlier, too.

Reply:
Nice catch. I wonder why proptype validation in the init of activitiesEditorRedux didn't catch this